src/uwtools/tests/test_logging.py

```diff
@@ -50,3 +50,13 @@ def test_setup_logging_verbose():
             format=ANY,
             level=logging.DEBUG,
         )
+
+
+def test_use_logger():
+    with patch.object(uwtools.logging, "log", uwtools.logging._Logger()):
+        # Initially, uwtools logging uses the Python root logger:
+        assert uwtools.logging.log.logger == logging.getLogger()
+        # But the logger can be swapped to use a logger of choice:
+        test_logger = logging.getLogger("test-logger")
+        uwtools.logging.use_logger(test_logger)
```

Comment:
So the user who plans to use their own logger and our API would just call this once they set up their own logger and then don't have to worry about it again?